Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -285,6 +291,8 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
         }()
 
         titleView.content = (title: title, subtitle: subtitle)
+        
+        channelAvatarView.content = (channelController.channel, channelController.client.currentUserId)
```

Comment:
Maybe we should rename the method from `updateNavigationTitle` to `updateNavigationBarContent`?